packages/legend-studio/src/components/editor/edit-panel/service-editor/ServiceEditor.tsx

```diff
@@ -273,7 +274,7 @@ const ServiceGeneralEditor = observer(() => {
           Owners
         </div>
         <div className="panel__content__form__section__header__prompt">
-          Specifies who can manage and operate the service
+          Specifies who can manage and operate the service. Requires two owners.
```

Comment:
--> `Specifies who can manage and operate the service (requires minimum 2 owners)`